apps/web/package.json

```diff
@@ -38,7 +38,7 @@
     "node": ">=18"
   },
   "dependencies": {
-    "@bermuda/sdk": "git+ssh://git@github.com:BermudaBay/sdk",
+    "@bermuda/sdk": "git+ssh://git@github.com:BermudaBay/sdk#sync-stxc-safe-pt3",
```

Comment:
Marker that this branch pinning can be removed once we merged the other PRs. We should then also re-generate the `yarn.lock` file.